app/src/org/commcare/utils/MediaUtil.java

```diff
@@ -257,7 +267,7 @@ private static Bitmap getBitmapScaledByTargetOrContainer(String imageFilepath,
         double widthScaleDownFactor =  (double)boundingWidth / originalWidth;
         // Choosing the larger of the scale down factors, so that the image still fills the entire
         // container
-        double dominantScaleDownFactor = Math.max(heightScaleDownFactor, heightScaleDownFactor);
+        double dominantScaleDownFactor = Math.max(widthScaleDownFactor, heightScaleDownFactor);

```

Comment:
Ugh yes thank you for catching. This was the part of this code I refactored more recently, so it at least hasn't been around for long.